src/components/topListingBadge/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC, PropsWithChildren } from 'react';
+import { Badge, Box } from '@chakra-ui/react';
+
+import { Grid, GridItem } from '../grid';
+
+type Props = {
+  aspectRatio?: number;
+};
+
+export const TopListingBadge: FC<PropsWithChildren<Props>> = ({
+  children,
+  aspectRatio,
+}) => {
+  return (
+    <Grid>
+      <GridItem gridColumn={1} gridRow={1}>
+        {aspectRatio ? <Box css={{ aspectRatio }}>{children}</Box> : children}
```

Comment:
is there a reason we cannot keep the `<AspectRatio />` component that it uses now? It doesn't seem to do anything on the storybook

Reply:
we can check it together if you want once you are back. This version is not working unfortunately :/